src/main/java/com/minecolonies/coremod/colony/buildings/BuildingDeliveryman.java

```diff
@@ -104,13 +120,27 @@ public View(final ColonyView c, final BlockPos l)
         @Override
         public Window getWindow()
         {
-            return new WindowHutDeliveryman(this);
+            return new WindowHutWorkerPlaceholder<>(this, DELIVERYMAN);
         }
 
         @Override
         public void deserialize(@NotNull final ByteBuf buf)
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this method to simply inherit it. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1185)